src/main/java/me/kodysimpson/simpapi/utility/builders/ItemBuilder.java

```diff
@@ -0,0 +1,192 @@
+     *
+     * @return The cloned instance.
+     */
+    public ItemBuilder clone() {
+        return new ItemBuilder(is);
+    }
+
+    /**
+     * Change the durability of the item.
+     *
+     * @param dur The durability to set it to.
+     */
+    public ItemBuilder setDurability(short dur) {
+        is.setDurability(dur);
+        return this;
+    }
+
+    /**
+     * Set the displayname of the item.
+     *
+     * @param name The name to change it to.
+     */
+    public ItemBuilder setName(String name) {
+        ItemMeta im = is.getItemMeta();
+        im.setDisplayName(ChatColor.translateAlternateColorCodes('&', name));
+        is.setItemMeta(im);
+        return this;
+    }
+
+    /**
+     * Add an unsafe enchantment.
+     *
+     * @param ench  The enchantment to add.
+     * @param level The level to put the enchant on.
+     */
+    public ItemBuilder addUnsafeEnchantment(Enchantment ench, int level) {
+        is.addUnsafeEnchantment(ench, level);
+        return this;
+    }
+
+    /**
+     * Remove a certain enchant from the item.
+     *
+     * @param ench The enchantment to remove
+     */
+    public ItemBuilder removeEnchantment(Enchantment ench) {
+        is.removeEnchantment(ench);
+        return this;
+    }
+
+    public ItemBuilder addItemFlag(ItemFlag flag) {
+        ItemMeta im = is.getItemMeta();
+        im.addItemFlags(flag);
+        is.setItemMeta(im);
+        return this;
+    }
+
+    public ItemBuilder removeItemFlag(ItemFlag flag) {
+        ItemMeta im = is.getItemMeta();
+        if (im.hasItemFlag(flag)) {
+            im.removeItemFlags(flag);
+            is.setItemMeta(im);
+        }
+        return this;
+    }
+
+    /**
+     * Add an enchant to the item.
+     *
+     * @param ench  The enchant to add
+     * @param level The level
+     */
+    public ItemBuilder addEnchant(Enchantment ench, int level) {
+        ItemMeta im = is.getItemMeta();
+        im.addEnchant(ench, level, true);
+        is.setItemMeta(im);
+        return this;
+    }
+
+    /**
+     * Add multiple enchants at once.
+     *
+     * @param enchantments The enchants to add.
+     */
+    public ItemBuilder addEnchantments(Map<Enchantment, Integer> enchantments) {
+        is.addEnchantments(enchantments);
+        return this;
+    }
+
+    /**
+     * Sets infinity durability on the item by setting the durability to Short.MAX_VALUE.
+     */
+    public ItemBuilder setInfinityDurability() {
+        is.setDurability(Short.MAX_VALUE);
+        return this;
+    }
+
+
+    /**
+     * Sets the armor color of a leather armor piece. Works only on leather armor pieces.
+     *
+     * @param color The color to set it to.
+     */
+    public ItemBuilder setLeatherArmorColor(Color color) {
+        try {
+            LeatherArmorMeta im = (LeatherArmorMeta) is.getItemMeta();
+            im.setColor(color);
+            is.setItemMeta(im);
+        } catch (ClassCastException expected) {
+        }
```

Comment:
There is no reason this needs to be in a try/catch. Make a static final `Set` of `Material`s that this method will work with. Check if the set contains the current instance's item's type, return early if it doesn't.